xapian-core/tests/harness/backendmanager_remotetcp.cc

```diff
@@ -404,3 +404,22 @@ BackendManagerRemoteTcp::clean_up()
     }
 #endif
 }
+
+void
+BackendManagerRemoteTcp::killall_server()
+{
+    for (unsigned i = 0; i < sizeof(pid_to_fd) / sizeof(pid_fd); ++i) {
```

Comment:
This needs wrapping with `#ifdef HAVE_FORK` (currently the CI appveyor builds fail because of this).

It'd be good to implement this for the `__WIN32__` case too - I think that would require noting down `procinfo.hProcess` above so we can use that to kill the child process here with `TerminateProcess()` or something.  Perhaps that's better left until we have per-server killing, but while it's not implemented we should throw an exception for that case rather than quietly doing nothing.